### DISCUSSION TITLE
Erro: No mapping for the unicode chacacter exists in the

### DISCUSSION THREAD

**Discussion by gleikocarmelo** [viniciussanchez/RESTRequest4Delphi#172] [Q&A]
Pessoal estou tendo esse erro ao realizar um Get onde no Content tenha alguma palavra com acento.
Utilizo banco: MySQL, Delphi 10.3.3
Obs: Realizando a consulta pelo Postman da certo normalmente, somente pelo Delphi que não.

**Comment by viniciussanchez** [comment]
Da uma olhada no canal no Youtube do Alexandre Magno, tem bastante conteúdo lá onde ele mostra na prática resolvendo esse problema. Ou também manda uma mensagem lá no nosso grupo do Telegram, que o pessoal que já passou por isso vai te ajudar, inclusive o Magno está por lá...
